tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec

```diff
@@ -10,27 +10,13 @@
         <icon>content\images\logo.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>
-            Assemblies required to start a DynamoModel and execute DesignScript code. Contains the following files:
-            1) DynamoApplications.dll
-            2) DynamoCore.dll
-            3) DynamoInstallDetective.dll
-            4) DynamoShapeManager.dll
-            5) DynamoUtilities.dll
-            6) ProtoCore.dll
-            7) DesignScriptBuiltin.dll
-            8) VMDataBridge.dll
-            9) DSCPython.dll
-            10) DynamoPackages.dll
+            Assemblies required to reference core APIs from Dynamo.
         </description>
         <summary>This package contains the core assemblies for Dynamo.</summary>
         <copyright>Copyright Autodesk 2023</copyright>
         <dependencies>
-            <!--TODO - why do we reference nunit from dynamo core? This is an odd partial collection of dependencies to mark.-->
             <group targetFramework="$TargetFramework$">
-                <dependency id="NUnit" version="3.13.3"/>
                 <dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="$Version$"/>
-                <dependency id="Newtonsoft.Json" version="13.0.1"/>
```

Comment:
do you think this will cause problems?

Reply:
Yeah, removing it would probably cause build issues for some consumers.
NewtonSoft types still seem to be publicly exposed "near" core Dynamo APIs. I will add a task to mark those as obsolete...or move them somewhere more hidden.

Reply:
yeah, it's going to be a big lift to get rid of these types as we require users to use those attributes, perhaps if those attributes are also now used by system.text.json we can switch it out without breaking previous builds - but needs to be determined. It was something I was hopeful we could do for 3.0, but no dice.